ner_v2/detectors/pattern/phone_number/phone_number_detection.py

```diff
@@ -101,7 +101,19 @@ def detect_entity(self, text, **kwargs):
         """
         self.text = " " + text.lower().strip() + " "
         self.phone, self.original_phone_text = [], []
+
         for match in phonenumbers.PhoneNumberMatcher(self.text, self.country_code, leniency=0):
+            try:
+                national_number_len = len(str(match.number.national_number))
+
+                # Get the national number and check its length is below 8 (including contry code) and \
+                # Exclude numbers that are too short to be a valid phone number (e.g., ticket numbers)
+                if national_number_len < 8:
+                    self.original_phone_text.append(self.text)
+                    continue
+            except Exception:
+                pass
```

Comment:
We do not "pass" on exception. If this code is going to raise exceptions they should be logged, if not remove the try catch.

Reply:
@rootAvish - Thanks, it's added via struct log.